src/garage/tf/models/cnn_mlp_merge_model.py

```diff
@@ -0,0 +1,141 @@
+            hidden layer. Default parameter points to second layer from the
+            end.
+        name (str): Model name, also the variable scope.
+        padding (str): The type of padding algorithm to use,
+            either 'SAME' or 'VALID'.
+        max_pooling (bool): Boolean for using max pooling layer or not.
+        pool_shapes (tuple[int]): Dimension of the pooling layer(s). For
+            example, (2, 2) means that all the pooling layers have
+            shape (2, 2).
+        pool_strides (tuple[int]): The strides of the pooling layer(s). For
+            example, (2, 2) means that all the pooling layers have
+            strides (2, 2).
+        cnn_hidden_nonlinearity (callable): Activation function for
+            intermediate dense layer(s) in the CNN. It should return a
+            tf.Tensor. Set it to None to maintain a linear activation.
+        cnn_hidden_w_init (callable): Initializer function for the weight of
+            intermediate dense layer(s) in the CNN. Function should return a
+            tf.Tensor.
+        cnn_hidden_b_init (callable): Initializer function for the bias of
+            intermediate dense layer(s) in the CNN. Function should return a
+            tf.Tensor.
+        hidden_nonlinearity (callable): Activation function for intermediate
+            dense layer(s) in the MLP. It should return a tf.Tensor. Set it to
+            None to maintain a linear activation.
+        hidden_w_init (callable): Initializer function for the weight
+            of intermediate dense layer(s) in the MLP. The function should
+            return a tf.Tensor.
+        hidden_b_init (callable): Initializer function for the bias
+            of intermediate dense layer(s) in the MLP. The function should
+            return a tf.Tensor.
+        output_nonlinearity (callable): Activation function for output dense
+            layer in the MLP. It should return a tf.Tensor. Set it to None
+            to maintain a linear activation.
+        output_w_init (callable): Initializer function for the weight
+            of output dense layer(s) in the MLP. The function should return
+            a tf.Tensor.
+        output_b_init (callable): Initializer function for the bias
+            of output dense layer(s) in the MLP. The function should return
+            a tf.Tensor.
+        layer_normalization (bool): Bool for using layer normalization or not.
+    """
+
+    def __init__(self,
+                 filter_dims,
+                 num_filters,
+                 strides,
+                 hidden_sizes=[256],
+                 action_merge_layer=-2,
+                 name=None,
+                 padding='SAME',
+                 max_pooling=False,
+                 pool_strides=(2, 2),
+                 pool_shapes=(2, 2),
+                 cnn_hidden_nonlinearity=tf.nn.relu,
+                 cnn_hidden_w_init=tf.glorot_uniform_initializer(),
```

Comment:
Please use the corresponding functions in TF 2.1

Reply:
fixed